plugins/kubernetes/test/models/kubernetes/usage_limit_test.rb

```diff
@@ -79,4 +79,17 @@ def create_limit(attributes = {})
       Kubernetes::UsageLimit.most_specific(project, deploy_group).must_equal found
     end
   end
+
+  describe "#priority" do
+    let(:limit) { create_limit(scope: environments(:staging)) }
+
+    it "is high with project" do
```

Comment:
aren't there other types to check too?

Reply:
usage limits only have project or are global